kong/plugins/aws-lambda/handler.lua

```diff
@@ -83,7 +83,11 @@ function AWSLambdaHandler:access(conf)
     return responses.send_HTTP_INTERNAL_SERVER_ERROR(err)
   end
 
-  ngx.status = res.status
+  if headers['X-Amzn-Function-Error'] == 'Unhandled' and conf.unhandled_response ~= -1 then
```

Comment:
I was about to mention this as well. When adding a new property in the plugin configuration, one needs to do one of those two things:
- create a migration for both C* and Postgres that adds the property (alongside its default) in all the existing rows containing this plugin configuration
- handle the possibility that said configuration property is `nil` in the Lua logic

We historically used to prefer the former approach, to stay consistent with previous cases where a property was added to an existing plugin, and to follow the philosophy that the configuration schema is the source of truth, and a `required` field is guaranteed to be present. Lately however, I've been more and more of a fan of the second approach, since we avoid adding migrations to the projects, and it allows us to ship new features into our minor releases. If you retrieve the default from the schema when `conf.unhandled_response` is `nil`, I think you should be just fine :)